src/1-theory/10-async.adoc

```diff
@@ -0,0 +1,407 @@
+= Asynchronous Paradigms
+
+So far the discussion of these paradigms has probably seemed rather similar,
+they are all following a request and response structure. Getting a response from
+a request is one way to do things, but there are lots of situations where a
+client might expect more than one response, or a response with more information
+later as more work happens on the server side.
+
+// First, some terminology!
+
+// **Synchronous** -
+// **Asynchronous** -
+// **Concurrency** -
+// **Parallelagram** -
+
+// push v pull
+
+== Client-Server "Real-Time" Models
+
+Let's have a quick run through of some of the common models for doing
+asynchronous stuff. Even thought you might not want to use some of these, it is
+good to know what they are, even if that is only so you can recommend not doing
+it when somebody else brings them up in a planning meeting.
+
+=== Short Polling
+
+The first approach that often gets mentioned is short polling, which is the
+client-server equivalent of a child in the backseat of the car asking "are we
+there yet? are we there yet? are we there yet?". The client is making the same
+GET request to the same resource every few seconds, waiting for some sort of
+change in the answer.
+
+[quote,sp3c1,https://stackoverflow.com/questions/4642598/short-polling-vs-long-polling-for-real-time-web-applications]
+----
+00:00:00 C-> Is the cake ready?
+00:00:01 S-> No, wait.
+00:00:01 C-> Is the cake ready?
+00:00:02 S-> No, wait.
+00:00:02 C-> Is the cake ready?
+00:00:03 S-> Yeah. Have some lad.
```

Comment:
```suggestion
00:00:03 S-> Yeah. Have some.
```
I think it will be more friendly towards all the audiences if you drop the lad. Doesn't add anything to this example anyway 🤷‍♀️

Reply:
Absolutely. This is a British thing, when I saw this I heard it in a yorkshire/midlands accent and with the guys picture next to it it all seemed to make sense. Definitely don't want to purpatrate the "everyone reading this is a man" thing though, thank you!